controllers/depresolver/depresolver_spec.go

```diff
@@ -24,15 +25,21 @@ func (dr *DependencyResolver) ResolveGslbSpec(ctx context.Context, gslb *k8gbv1b
 		if strategy.SplitBrainThresholdSeconds == 0 {
 			strategy.SplitBrainThresholdSeconds = predefinedStrategy.SplitBrainThresholdSeconds
 		}
-		dr.errorSpec = dr.validateSpec(strategy)
+		dr.errorSpec = dr.validateStrategy(strategy)
 		if dr.errorSpec == nil {
 			dr.errorSpec = dr.client.Update(ctx, gslb)
 		}
+		// check for missing .spec.ingress.host.http
```

Comment:
Imo this code is self-explanatory, no comments needed.
Or you can consider extracting this code into something like `ValidateIngressSpec` method, in this case you definitely don't need this comment :)